src/pages/Signup.js

```diff
@@ -0,0 +1,49 @@
+import "../styles/auth.css";
+import {NavMenu} from "../components/NavMenu";
+import {Link} from "react-router-dom";
+const Signup = () => {
+    return (
+        <>
+        <NavMenu />
+        <section className="form-ctn br-md">
+        <form>
+            <h2 className="text-center mg-bottom-md">Signup</h2>
+            <div className="form-control">
+                <label htmlFor="first-name" className="fw-bold">First name</label>
+                <input type="text" name="first-name" placeholder="first name" />
```

Comment:
- here you forgot to add ```id``` attribute.
```suggestion
                <input type="text" name="first-name" id="first-name" placeholder="first name" />
```